src/sagemaker/jumpstart/enums.py

```diff
@@ -75,3 +75,34 @@ class JumpStartTag(str, Enum):
     INFERENCE_SCRIPT_URI = "aws-jumpstart-inference-script-uri"
     TRAINING_MODEL_URI = "aws-jumpstart-training-model-uri"
     TRAINING_SCRIPT_URI = "aws-jumpstart-training-script-uri"
+
+
+class EnvVariableUseCase(str, Enum):
+    """Enum class for the different use cases for hosting environment variables."""
+
+    # These environment variables are required when calling
+    # the low-level `SageMaker.CreateModel` API.
+    # See: https://docs.aws.amazon.com/sagemaker/latest/APIReference/API_CreateModel.html
+    CREATE_MODEL_API = "create_model_api"
```

Comment:
If we introduce a `scope` in the future for the `environment_variables` utility, how about defining two enum members which would be more self-explanatory:
- `AWS_SDK`
- `SAGEMAKER_PYTHON_SDK`